README_validations.md

```diff
@@ -26,7 +26,7 @@ For OSCAL XML, this schema syntax is [XML Schema Definition (XSD)](https://www.w
 
 Each tool will have its own way to provide for validation. Bindings or configurations to link documents to schemas can be maintained either inside or outside the documents themselves; many tools offer several ways to do this.
 
-Some tools support self-contained validation functionality without full-fledged application frameworks or development environments, with tools like [`xmllint`](http://xmlsoft.org/xmllint.html) offering command-line interfaces. Similarly, you can use the [`ajv-cli`](https://github.com/ajv-validator/ajv-cli) for JSON.
+Some tools support self-contained validation functionality without full-fledged application frameworks or development environments, with tools like [`xmllint`](https://gitlab.gnome.org/GNOME/libxml2/) offering command-line interfaces. Similarly, you can use the [`ajv-cli`](https://github.com/ajv-validator/ajv-cli) for JSON.
```

Comment:
Non-block comment for @david-waltermire-nist to understand what I encountered: xmlsoft.org now seems to be dead, that is what is breaking the Markdown link validation. The redirect target is bad. The HTTPS cert is not properly configured for the correct sub-domain (for *.gnome.org, not *pages.gitlab.gnome.org) and going HTTP forces HTTPS redirect to the bad target. So, I will go to the GitLab project URL, not the rendered wiki pages hosted from GitLab.

```sh
$ curl -I http://xmlsoft.org/xmllint.html 
HTTP/1.1 302 Found
Date: Thu, 25 Aug 2022 14:18:26 GMT
Server: Apache/2.2.15 (CentOS)
Location: https://gnome.pages.gitlab.gnome.org/libxml2/xmllint.html
Connection: close
Content-Type: text/html; charset=iso-8859-1

$ curl -I http://gnome.pages.gitlab.gnome.org/libxml2/xmllint.html
HTTP/1.1 302 Found
content-length: 0
location: https://gnome.pages.gitlab.gnome.org/libxml2/xmllint.html
cache-control: no-cache
```

Reply:
Per weekly team sync, will review with Dave. There are concerns that GitLab landing page README doesn't really give command instructions or any helpful end user dev docs. Will work it out with Dave when we touch base.

Reply:
Per conversation with @david-waltermire-nist, this seems to be a known issue and impacts all the GitLab pages hosted on gitlab.gnome.org because you cannot use LetsEncrypt (or any TLS certificate authority for that matter) to have multiple-wildcards in the SAN and [GNOME Infrastructure Team has marked this WONTFIX in their issue board](https://gitlab.gnome.org/Infrastructure/GitLab/-/issues/395). Going to use the popular die.net for its indexed man page collection, and remove the download link.